hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -57,6 +58,7 @@ public final class OmKeyInfo extends WithParentObjectId
 
   private static final Codec<OmKeyInfo> CODEC_TRUE = newCodec(true);
   private static final Codec<OmKeyInfo> CODEC_FALSE = newCodec(false);
+  private static final String COMMITTED_KEY_DELETED_FLAG = "CKDEL";
```

Comment:
Add a javadoc. Tweak as you see fit


```suggestion
  /**
   * Metadata key flag to indicate whether a deleted key was a committed key.
   * The flag is set when a key is deleted from AOS but still held in a snapshot
   * to help with accurate bucket quota usage calculation.
   */
  private static final String COMMITTED_KEY_DELETED_FLAG = "CKDEL";
```